.github/drafts/stale.yml

```diff
@@ -10,13 +10,16 @@ jobs:
       - uses: actions/stale@main
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-pr-message: 'Hello there,
-          we noticed that this PR has been inactive for a while now. If there are any changes which are suitable for our repository,
-          we would love to have it. It would be great if you can continue with this PR, but if you cannot, we might fork your changes and merge the changes ourselves.
+          stale-pr-message: "Hello there,
+          we have noticed that this PR has been inactive for a while now. If there are any changes which are suitable for our repository,
+          we would love to have it. It would be great if you can continue with this PR, but if you can't, we might fork your changes and merge the changes ourselves.
```

Comment:
```suggestion
          we would love to have it. It would be great if you can continue with this PR, but if you can't, we might fork your changes and merge them ourselves.
```